druid-shell/src/platform/windows/util.rs

```diff
@@ -144,6 +146,8 @@ type CreateDXGIFactory2 =
 #[allow(non_snake_case)] // For member fields
 pub struct OptionalFunctions {
     pub GetDpiForSystem: Option<GetDpiForSystem>,
+    pub GetDpiForWindow: Option<GetDpiForWindow>,
+    pub SetProcessDpiAwarenessContext: Option<SetProcessDpiAwarenessContext>,
```

Comment:
I was going to suggest cleaning this up a bit, but now I'm not sure. We don't currently use `GetDpiForMonitor`, but I think it is the function we want to use for compatibility on Windows 8.1. Similar for SetProcessDpiAwareness.